packages/internal/src/build/babel/common.ts

```diff
@@ -0,0 +1,14 @@
+import type { TransformOptions } from '@babel/core'
+
+export interface RequireHookOptions extends TransformOptions {
```

Comment:
Does babel not have types for the register hook?

Reply:
Nope... weird eh?